packages/matter.js/src/behavior/internal/Reactors.ts

```diff
@@ -211,6 +211,10 @@ class ReactorBacking<T extends any[], R> {
         if (!(error instanceof Error)) {
             error = new Error(`${error}`);
         }
+        if (error instanceof MatterError) {
```

Comment:
I'm wondering if we should be differentiating between MatterError and other errors here.  Seems like if we catch InternalError or ImplementationError for example it should be handled the same way as a bare Error.  Should we just allow all errors through like it used to work?

Reply:
So you mean rethrowing all errors? Ok we can do that and see if it gives issues ... I just comment out here for "reference" if we need to re-add